src/main/java/core/basesyntax/service/ReaderService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface ReaderService {
+    StringBuilder read();
```

Comment:
it will be better if method will return String(so you don`t have to convert int String in FruitServiceImpl)